lib/data/content/chapter-4.yml

```diff
@@ -1,19 +1,22 @@
-blockquote_text: "Although some ecoregions contain more biodiversity than others, all ecoregions require at least some form of protection in order for the world’s protected and conserved areas to capture a representative sample of species and ecosystems. There therefore remains a clear need to improve ecological representation, and attention is particularly needed in ecoregions that lack protection entirely, "
-content_3: "such as the Angolan montane forest-grassland mosaic ecoregion, Cargados Carajos/Tromelin Island ecoregion, and Eastern Anatolian deciduous forests ecoregion. Ways of increasing ecological representation include greater use of systematic conservation planning (Box 5), and consideration and wider uptake of nationally-differentiated targets (Box 6). "
-content_4: "<sup>1</sup> Ecological units of land, ocean or freshwater that share similar and geographically-distinct biological characteristics (Olson <em>et al.,</em> 2001; Spalding <em>et al.,</em> 2012, 2007)."
+content_2: "However, many ecoregions remain under-protected. The 365 terrestrial ecoregions with at least 17% coverage by protected areas and OECMs represent fewer than half (44.5%) of all terrestrial ecoregions. Similarly, the 110 marine ecoregions with at least 10% coverage represent only 47.4% of such ecoregions. This drops further to 10.8% in pelagic provinces, which are considerably larger and fall largely outside national jurisdiction. Methodologies and datasets for assessing coverage of freshwater ecoregions vary, but studies agree they are less than 22% protected in total (Bastin <em>et al.,</em> 2019; UNEP-WCMC <em>et al.,</em> 2018)."
+blockquote_text: "Although some ecoregions contain more biodiversity than others, all ecoregions require at least some form of protection in order for the world’s protected and conserved areas to capture a representative sample of species and ecosystems."
+content_3: "There therefore remains a clear need to improve ecological representation, and attention is particularly needed in ecoregions that lack protection entirely, such as the Angolan montane forest-grassland mosaic ecoregion, Cargados Carajos/Tromelin Island ecoregion, and Eastern Anatolian deciduous forests ecoregion. Ways of increasing ecological representation include greater use of systematic conservation planning (Box 5), and consideration and wider uptake of nationally-differentiated targets (Box 6). "
+content_4: "<sup>1</sup> Ecological units of land, ocean or freshwater that share similar and geographically-distinct biological characteristics (Olson <em>et al.,</em> 2001; Spalding <em>et al.,</em> 2012, 2007).
+
+<sup>2</sup> Antarctic, Rock and Ice and Lake ecoregions have been excluded.
```

Comment:
Got an error here. This second footnote isn't appearing.

<img width="1104" alt="image" src="https://user-images.githubusercontent.com/56026099/118279691-5570a480-b4c3-11eb-9f01-5ef3f4a86089.png">

Reply:
Sorry, my bad. It is there but in the same block